kombu/transport/kafka.py

```diff
@@ -0,0 +1,336 @@
+"""
+kombu.transport.kafka
```

Comment:
Make that 3 options to choose from. The members of this discussion missed confluent-kafka-client, which seems to now lead the group in terms of performance.

see this blog post: http://activisiongamescience.github.io/2016/06/15/Kafka-Client-Benchmarking/